DATA/common/setenv.sh

```diff
@@ -15,7 +15,10 @@ LIST_OF_GLORECO="ITSTPC,TPCTRD,ITSTPCTRD,TPCTOF,ITSTPCTOF,MFTMCH,PRIMVTX,SECVTX,
 
 # Detectors used in the workflow / enabled parameters
 if [ -z "${WORKFLOW_DETECTORS+x}" ] || [ "0$WORKFLOW_DETECTORS" == "0ALL" ]; then export WORKFLOW_DETECTORS=$LIST_OF_DETECTORS; fi
-if [ -z "${WORKFLOW_DETECTORS_QC+x}" ] || [ "0$WORKFLOW_DETECTORS_QC" == "0ALL" ]; then export WORKFLOW_DETECTORS_QC=$WORKFLOW_DETECTORS; fi
+if [ -z "${WORKFLOW_DETECTORS_QC+x}" ] || [ "0$WORKFLOW_DETECTORS_QC" == "0ALL" ]; then
+    export WORKFLOW_DETECTORS_QC=$WORKFLOW_DETECTORS;
+    WORKFLOW_DETECTORS_QC+=",$LIST_OF_GLORECO"
```

Comment:
Why don't you just do:
```
if [ -z "${WORKFLOW_DETECTORS_QC+x}" ] || [ "0$WORKFLOW_DETECTORS_QC" == "0ALL" ]; then export WORKFLOW_DETECTORS_QC="$WORKFLOW_DETECTORS,$LIST_OF_GLORECO"; fi
```

Reply:
actually, the way you have it right now is actually wrong, since you export `WORKFLOW_DETECTORS_QC=$WORKFLOW_DETECTORS` and then you modify it only for the local shell.
If this is used in a subshell, it will loose the `$LIST_OF_GLORECO`

Reply:
Done. I thought like this it was clearer in case it was not ok ;-)